clients/banking/src/graphql/partner.gql

```diff
@@ -496,7 +496,6 @@ fragment CardListItem on Card {
   type
   name
   statusInfo {
-    __typename
```

Comment:
No need to change this

Reply:
we don't need it anymore. I put it in a PR yesterday to fix the feature I was working on